src/pipeline.py

```diff
@@ -44,7 +67,7 @@ def run_pipeline(start):
     # Check data file exists
```

Comment:
config["paths"] needs to be updated- pipeline currently stops here.

Reply:
It works if we add 

`    if HDFS_AVAILABLE:
        snapshot_path = config["hdfs_paths"]["snapshot_path"]
    else:
        snapshot_path = config["network_paths"]["snapshot_path"]`

Reply:
I think this is an old comment now because we fixed this.